app/models/agents/pushbullet_agent.rb

```diff
@@ -17,21 +17,15 @@ class Unauthorized < StandardError; end
 
     description <<-MD
       The Pushbullet agent sends pushes to a pushbullet device
-
```

Comment:
Those empty lines are needed, markdown does only creates a new line when the paragraphs are separated with an empty line.